tests/kuttl/tests/dataplane-deploy-global-service-test/00-dataplane-create.yaml

```diff
@@ -117,6 +117,22 @@ kind: OpenStackDataPlaneNodeSet
 metadata:
   name: edpm-compute-global
 spec:
+  baremetalSetTemplate:
```

Comment:
It's weird that we've to add defaulted fields when creating a CR. This is one of the reasons why I did not want hashes to be asserted in tests unless we're testing the use-cases for them and my feedback was ignored in https://github.com/openstack-k8s-operators/dataplane-operator/pull/725.